### ISSUE TITLE
After a successful IPP payment, email receipt action brings up a printable receipt instead of email UI

### ISSUE THREAD

**Issue by jaclync** [woocommerce/woocommerce-ios#14512]
**Describe the bug**

After collecting payment with card reader successfully, tapping the CTA to email receipt doesn't present UI to email. A printable receipt view is presented instead. I can reproduce this in both iPhone and iPad devices.

**To Reproduce**

Please test this on a physical device with email capability.

- Switch to a store eligible for IPP
- Go to the orders tab
- Tap + to create an order, add a custom amount or some purchasable products
- Tap `Collect Payment` > `Card Reader` and proceed with the card reader connection and payment with a test card --> a success modal should be shown
- Tap `Email receipt` --> after dismissal, notice that the printable receipt view is shown instead of some UI to send an email

**Screenshots**

https://github.com/user-attachments/assets/67dd2dd6-e645-4a69-b605-dccb77eb589f

**Expected behavior**

**Isolating the problem (mark completed items with an [x]):**
- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**Mobile Environment**
Please include:
- Device: iPad Pro 11in 3rd gen / iPhone 13 Pro
- iOS version: iOS 17.5
- WooCommerce iOS version: 21.1/21.2

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by staskus** [comment]
@jaclync, I think this functionality is expected and was implemented for the backend receipts project. @iamgabrielma should have more context to close this or keep it open.

**Comment by jaclync** [comment]
> I think this functionality is expected and was implemented for the backend receipts project. @iamgabrielma should have more context to close this or keep it open.

Did you mean that it's expected to show the receipt printing view from the `Email receipt` action, which is the same as the action when tapping `Print receipt`? If so, maybe we can remove `Email receipt` CTA to avoid confusion. I was expecting an email form with the receipt content from the `Email receipt` action.

**Comment by staskus** [comment]
@jaclync Yes, as I understood when looking at the code comments, those actions were deliberately made to make the same action. It was done to have a parity with the "local receipts" functionality. However, I agree with you that the "Email receipt" option could be removed to avoid confusion.
